crates/release_plz/tests/all/release_pr.rs

```diff
@@ -19,6 +27,25 @@ async fn release_plz_should_set_custom_pr_details() {
     let opened_prs = context.opened_release_prs().await;
     assert_eq!(opened_prs.len(), 1);
     assert_eq!(opened_prs[0].title, expected_title);
+    assert_eq!(
+        opened_prs[0].body.as_ref().unwrap().trim(),
+        format!(
+            r#"
+    === [0.1.0](https://localhost/{}/{}/releases/tag/v0.1.0) - 2024-10-16
+    Package: {} 0.1.0 -> 0.1.0
+    Changes:
+    ### Other
+
+- add config file
+- cargo init
+- Initial commit
```

Comment:
why isn't this well indented? 🤔

Reply:
`release.changelog` is a block of text, indentation applies only to the first line